docs/ref/settings.txt

```diff
@@ -3260,6 +3277,7 @@ Security
   * :setting:`CSRF_COOKIE_NAME`
   * :setting:`CSRF_COOKIE_PATH`
   * :setting:`CSRF_COOKIE_SECURE`
+  * :setting:`CSRF_HEADER_NAME`
```

Comment:
Aphabetical sorting please (and for the section about the setting above as well)